CHANGELOG.next.asciidoc

```diff
@@ -43,7 +43,7 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - Removed deprecated Sophos UTM from Beats. Use the https://docs.elastic.co/integrations/sophos[Sophos] Elastic integration instead. {pull}38037[38037]
 - Introduce input/netmetrics and refactor netflow input metrics {pull}38055[38055]
 - Update Salesforce module to use new Salesforce input. {pull}37509[37509]
-
+- Remove dashboards from the revamped Salesforce module. {pull}39028[39028]
```

Comment:
We do not need this. Ultimately one changelog goes i.e., for module. No need for this?